MyPerfectOnboardingApplication/MyPerfectOnboarding.Api/Models/ListItem.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Web;
+
+namespace MyPerfectOnboarding.Api.Models
+{
+    [DataContract]
```

Comment:
You do not have to specify DataContract as you [do not specify in the json formatter to use it](https://docs.microsoft.com/en-us/aspnet/web-api/overview/formats-and-model-binding/json-and-xml-serialization). In default WebApi uses JSON.NET.